fw/http.c

```diff
@@ -1540,6 +1540,7 @@ __tfw_http_resp_fwd_stale(TfwHttpMsg *hmresp)
 
 	return sent;
 }
+ALLOW_ERROR_INJECTION(__tfw_http_resp_fwd_stale, NULL);
```

Comment:
I am not sure that it is ok to return NULL for error injection if this function really returns bool value